### PR TITLE
Done

### PR DESCRIPTION
**Pull request by Perchitti**

### CODE REVIEW

File: bin/tictactoe [hunk 1]
```diff
@@ -1,3 +1,23 @@
 #!/usr/bin/env ruby
```
Comment (kkterai) [inline bin/tictactoe]: Prefer to encapsulate CLI logic into a class. For example, the only code in your bin executable should be:

```ruby
#!/usr/bin/env ruby
       
require_relative '../config/environment'

TicTacToeCLI.new.call
```

And then you'd have a file: `lib/tic_tac_toe_cli.rb` with:

```ruby
class TicTacToeCLI
  def initialize
  end

  def call
   # Your game logic
  end
end
```

The code in your bin would generally just go in `call`.

File: lib/board.rb [hunk 2]
```diff
@@ -0,0 +1,50 @@
+require 'pry'
+
+class Board
+
+
+attr_accessor :cells
+
+
+def initialize
+  @cells = Array.new(9, " ")
```
Comment (kkterai) [inline lib/board.rb]: I would just call `self.reset!` here and delegate to that method to create the default empty board.

File: lib/game.rb [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+def won?
+    WIN_COMBINATIONS.detect do |combo|
+      @board.cells[combo[0]] == @board.cells[combo[1]] &&
+      @board.cells[combo[0]] == @board.cells[combo[2]] &&
+      @board.taken?(combo[0] + 1)
+    end
+  end
+
+  def current_player
+    @board.turn_count % 2 == 0 ? @player_1 : @player_2
```
Comment (kkterai) [inline lib/game.rb]: Use [`Integer#even?`](http://ruby-doc.org/core-1.8.7/Integer.html#method-i-even-3F) or [`Integer#odd?`](http://ruby-doc.org/core-1.8.7/Integer.html#method-i-odd-3F)

`@board.turn_count.even? ? @player_1 : @player_2`

File: lib/game.rb [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+
+def initialize(player_1 = Players::Human.new("X"), player_2 = Players::Human.new("O"), board = Board.new)
+  @board = board
+  @player_1 = player_1
+  @player_2 = player_2
+end
+
+
+def won?
+    WIN_COMBINATIONS.detect do |combo|
```
Comment (kkterai) [inline lib/game.rb]: Nice job using `#detect` here!

File: lib/players/human.rb [hunk 5]
```diff
@@ -0,0 +1,12 @@
+module Players
+  class Human < Player
+
+def move(input)
+  puts "Please enter 1-9:"
+  gets.strip
```
Comment (kkterai) [inline lib/players/human.rb]: Nice job using implicit returns!

**Comment by Perchitti** [comment]
Thank you Kenlyn

**Pull request closed without merging**